lib/enkf/enkf_main_jobs.cpp

```diff
@@ -76,6 +76,7 @@ void * enkf_main_dummy_JOB(void * self , const stringlist_type * args ) {
 
 
 void * enkf_main_ensemble_run_JOB( void * self , const stringlist_type * args ) {
+  printf("WARNING: The ENSEMBLE_RUN job is deprecated and will soon be removed!\n");
```

Comment:
Looks good, but perhaps we should put a date on when it will be removed? Guess standard is 1 year, but maybe end of this year?

Reply:
Dates are always scary, because then you suddenly have a warning saying this will be removed in 2012 😂 Also, I don’t expect these messages to hit the users. So we’ll have to make announcements in addition 👍